src/i18n/Localizable.xcstrings

```diff
@@ -22020,7 +22134,8 @@
       }
     },
     "Show_Scrollback_Buffer" : {
-      "extractionState" : "migrated",
+      "comment" : "is this one used as gerund or noun?",
```

Comment:
Verb

Reply:
deleted comment and used gerund, same reason for "Restart"